proto/cacheclient.proto

```diff
@@ -435,6 +438,16 @@ message _ListLengthResponse {
   }
 }
 
+message _ListRetainRequest {
+  bytes list_name = 1;
+  optional uint32 count = 2;
+  optional sint32 start_index = 3;
+  optional uint64 ttl_milliseconds = 4;
+  optional bool refresh_ttl = 5;
+}
+
+message _ListRetainResponse {}
```

Comment:
Removed the response, I don't think we really talked about it, and if we wanted to return the trimmed result.